common/src/chain/transaction/output/mod.rs

```diff
@@ -79,21 +79,21 @@ impl TxOutput {
             | TxOutput::LockThenTransfer(_, _, _)
             | TxOutput::StakePool(_)
             | TxOutput::DecommissionPool(_, _, _, _)
-            | TxOutput::ProduceBlockFromStake(_, _, _) => false,
+            | TxOutput::ProduceBlockFromStake(_, _) => false,
             TxOutput::Burn(_) => true,
         }
     }
 }
 
 impl TxOutput {
-    pub fn value(&self) -> OutputValue {
+    pub fn value(&self) -> Option<OutputValue> {
```

Comment:
This change bothers me because making it `Option` doesn't provide enough information for the outside code on how to treat it. Is None equivalent to No-op, zero or an Err?

Particularly in `mempool/try_get_fee.rs` it seems unclear what to do with it and very plausible just to ignore it with unclear consequences for the fee calculation. 

I tend to think that this method should go and it will be impossible to get output value without the context (i.e. without pos_accounting information)

Reply:
@TheQuantumPhysicist

Reply:
Yep. This should go. Just like the destination. I just kept it because I didn't want to introduce functionality change. Now it's time to remove it.